modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -4902,18 +4902,18 @@ class CV_EXPORTS LineIterator
     connects the given two points.
     The line will be clipped on the image boundaries.
 
-    @param img Underlying image.
+    @param src Underlying image.
     @param pt1 First endpoint of the line.
     @param pt2 The other endpoint of the line.
     @param connectivity Pixel connectivity of the iterator. Valid values are 4 (iterator can move
     up, down, left and right) and 8 (iterator can also move diagonally).
     @param leftToRight If true, the line is traversed from the leftmost endpoint to the rightmost
     endpoint. Otherwise, the line is traversed from \p pt1 to \p pt2.
     */
-    LineIterator( const Mat& img, Point pt1, Point pt2,
```

Comment:
what is the point of renaming `img` argument to `src`?
It is unnecessary change

Reply:
before PR there was an attribute Mat img in LineIterator. Many warning were generated parameter img hide  attribute . i change parameter name. attribute img was unused in class. I delete it after first or second review.
After two changes it is now unnecessary